src/components/families/FamilyDetailsSidebar.tsx

```diff
@@ -80,6 +87,10 @@ const FamilyDetailsSidebar = ({
               {family.parent.information[parentField.id]}
             </Typography>
           ))}
+          <Typography variant="body2" className={classes.heading}>
+            Notes
+          </Typography>
+          <TextField value={family.notes} fullWidth variant="filled" />
```

Comment:
getting a linter error on `family.notes` - since it's referenced using an enum on the type, we should call `family[DefaultFieldKey.NOTES]` instead!